zwave_js_server/model/duration.py

```diff
@@ -17,12 +17,16 @@ class DurationDataType(TypedDict, total=False):
 class Duration:
     """Duration class."""
 
-    data: DurationDataType
+    data: DurationDataType | Literal["unknown", "default"]
```

Comment:
Does this mean that the unit in `DurationDataType` can only have seconds or minutes as values?

Reply:
yup good call

Reply:
reference: https://github.com/zwave-js/node-zwave-js/blob/master/packages/core/src/values/Duration.ts#L134